cmd/pulumi-test-language/tests.go

```diff
@@ -774,8 +774,16 @@ var languageTests = map[string]languageTest{
 					requireStackResource(l, err, changes)
 
 					require.Len(l, snap.Resources, 2, "expected 2 resource")
-					stack := snap.Resources[0]
-					require.Equal(l, resource.RootStackType, stack.Type, "expected a stack resource")
+
```

Comment:
We should TODO why were doing this. It's very suspect, and I'd like to try and fix it at some point.

Reply:
Done ✅

Reply:
takk